premake5.lua

```diff
@@ -43,7 +43,9 @@ app( 'Geno' )
 		}
 
 	filter 'system:linux'
+		exceptionhandling( "On" ) -- Need this for a temporary fix to memory errors in OutputWindow.cpp
```

Comment:
@Th3T3chn0G1t Do we still need this now that we don't use std::string in OutputWindow ?

Reply:
Test <@606227114932568075>

Reply:
Another test <@177136743768653834>